verify.go

```diff
@@ -15,7 +15,11 @@ import (
 	"github.com/replit/go-replidentity/paserk"
 )
 
-func verifyToken(token string, pubkey ed25519.PublicKey) ([]byte, error) {
+type verifier struct {
+	currentClaims *MessageClaims
```

Comment:
per the suggestion below, it might be better to have this be

```suggestion
	claims *MessageClaims
```

since it won't be possible to change them after verifying a chain.

Reply:
renamed